### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by jimbo4794dev**

### CODE REVIEW

**Review by Jimbo4794 — COMMENTED** [review]
approved for build

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# automation
\ No newline at end of file
+# automation
```
Comment (Jimbo4794) [inline README.md:1]: Line comment

**Review by Jimbo4794 — COMMENTED** [review]
Another comment

**Comment by Jimbo4794** [comment]
Raw comment

**Comment by Jimbo4794** [comment]
Build comment

**Comment by galasa-team** [comment]
Automatic triggering of build cancelled as user is not a member of the approved groups. Please ask an admin to review this PR and create a comment on the PR stating 'Approved for building'

**Pull request closed without merging**